### PR TITLE
Settings: Bring back version and copyright

### PR DESCRIPTION
**Pull request by guruz**
For owncloud/enterprise#2941

### CODE REVIEW

File: src/gui/generalsettings.ui [hunk 1]
```diff
@@ -279,8 +363,6 @@
   <tabstop>newExternalStorage</tabstop>
   <tabstop>showInExplorerNavigationPaneCheckBox</tabstop>
   <tabstop>crashreporterCheckBox</tabstop>
-  <tabstop>updateChannel</tabstop>
-  <tabstop>restartButton</tabstop>
```
Comment (guruz) [inline src/gui/generalsettings.ui]: i have no idea why this is like that. When i click to edit the tabsstops in designer they are shown?

File: src/libsync/theme.h [hunk 2]
```diff
@@ -203,6 +203,7 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
      * About dialog contents
      */
     virtual QString about() const;
+    virtual bool aboutShowCopyright() const;
```
Comment (ogoffart) [inline src/libsync/theme.h]: (I would have made this return a QString, so they can customize the name if they want (or empty to hide), I foresee that customer will want that.)

Comment (michaelstingl) [reply src/libsync/theme.h]: Use `APPLICATION_VENDOR` here?

Should be `ownCloud GmbH` in the unbranded?
https://github.com/owncloud/client/blob/v2.5.0/OWNCLOUD.cmake#L5

Comment (michaelstingl) [reply src/libsync/theme.h]: Yeah, even when `APPLICATION_VENDOR` is "Acme Corp LLC", copyright is still "ownCloud GmbH".

**Comment by guruz** [comment]
Fix on top 62252d0012c00dbba64dbe3210f104f7ee58434c

**Pull request merged**